packages/video_player/video_player/pubspec.yaml

```diff
@@ -27,7 +27,7 @@ dependencies:
   html: ^0.15.0
   video_player_android: ^2.3.5
   video_player_avfoundation: ^2.5.6
-  video_player_platform_interface: ">=6.1.0 <7.0.0"
+  video_player_platform_interface: ">=6.2.0 <7.0.0"
```

Comment:
This can just be `^6.2.0`.

Reply:
You're totally right, let me fix this so we don't propagate the ">=<" syntax further than strictly needed.